app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    all_cache = {}
+
+    def inner(*args) -> int:
+        if args in all_cache:
```

Comment:
* use `functools.wraps`
* what if the function takes the same `args`  but different `kwargs`?
* you must use `args` with `kwargs` as the key of `all_cache`